### ISSUE TITLE
[🐛 Bug]: addLocatorStrategy is missing inside attach

### ISSUE THREAD

**Issue by Nio-o** [webdriverio/webdriverio#7724]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

7.16.8

### Node.js Version

14.17.0

### Mode

Standalone Mode

### Which capabilities are you using?

```typescript
desiredCapabilities: {
  'goog:chromeOptions': {
    args: ['disable-web-security'],
  },
  browserName: 'chrome',
},
```

### What happened?

Wdio doesn't set addLocatorStrategy when using attach function, it is only for remote/multiremote
Is it expected behavior?

https://github.com/webdriverio/webdriverio/blob/f9eae7a6747b4bf55975a891f25643bba2165936/packages/webdriverio/src/index.ts#L75

### What is your expected behavior?

I expect that addLocatorStrategy is always set

### How to reproduce the bug.

I'm using hermione, so I use part of hermione code to show bug
https://github.com/gemini-testing/hermione/blob/a2f03022fa64cd3668e59d05917bb6ad377e82db/lib/browser/existing-browser.js#L131

Simplified example: just attach and check that addLocatorStrategy is undefined
`webdriverio.attach.then((browser) => console.log({browser})`

### Relevant log output

```typescript
-
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by christian-bromann** [comment]
Thanks for reporting!



Any contributions that resolves the bug are highly appreciated. Please take a look into our [contribution guidelines](https://webdriver.io/docs/contribute) and let us know if you have any questions. Cheers!

**Comment by Pawel1894** [comment]
Is this still an issue? If yes, can you assign it to me?

**Comment by Pawel1894** [comment]
Hyped for my first task :D

**Comment by Pawel1894** [comment]
@christian-bromann FYI this issue can be closed - released in **v8.19.0**

**Comment by erwinheitzman** [comment]
Thanks @Pawel1894 👍
